gamemode/modules/f1menu/cl_searchbox.lua

```diff
@@ -6,15 +6,18 @@ function PANEL:Init()
     self.BaseClass.Init(self)
     self.F1Down = true
     self:SetFont("DarkRPHUD2")
-    self:SetTextColor(Color(255,255,255,255))
-    self:SetCursorColor(Color(255,255,255,255))
 
     self.lblSearch = vgui.Create("DLabel", self)
     self.lblSearch:SetFont("DarkRPHUD2")
-    self.lblSearch:SetColor(Color(200, 200, 200, 200))
     self.lblSearch:SetText(DarkRP.getPhrase("f1Search"))
     self.lblSearch:SizeToContents()
     self.lblSearch:SetPos(5)
+
+    if GAMEMODE.Config.DarkRPSkin == "DarkRP" then
+        self:SetTextColor(Color(255,255,255,255))
+        self:SetCursorColor(Color(255,255,255,255))
+        self.lblSearch:SetColor(Color(200, 200, 200, 200))
+    end
```

Comment:
> (I've noticed myself that changing the properties of things in a SKIN table away from what the default derma skin has set can cause elements not using that skin to also be affected - can make things look very odd)

That should never happen. If it did then existing DarkRP SKIN fields would also have the issue and I haven't seen a complaint about it. If you found a way to reproduce this bug however, then it would definitely be worth reporting it.

> (changing the colour of your search boxes for one element is a tad extreme)

Addons, future DarkRP updates etc. could change this. The current transparent background for text entries currently is applied to _all_ DTextEntry elements with the DarkRP skin which is why a global colour change makes sense.

The only case I see it being a problem, because of the transparent background of the element, is if someone used a light background in the parent but then they are no longer using the DarkRP colour scheme (ie. they are using something like a hardcoded `SetBackgroundColor`/`SetBGColor`) so should be making their own skin (or even continue their hardcoded colour-setting decision by using `SetTextColor`). There would likely be conflicts anyway with a blue or red background like some other DarkRP elements used.